packages/cloudflare/src/entrypoints/image-service.ts

```diff
@@ -25,6 +25,7 @@ const service: ExternalImageService = {
 		}
 
 		const imageEndpoint = joinPaths(
+			// @ts-expect-error - Property 'env' does not exist on type 'ImportMeta'.ts(2339)
```

Comment:
I thought this was fixed with the update to the matching/newest vite version 🤔

Reply:
No this was by fixed by the `astro/client` reference